packages/react-composites/src/composites/common/styles/ParticipantContainer.styles.ts

```diff
@@ -94,12 +95,19 @@ export const peoplePaneContainerTokens: IStackTokens = {
 /**
  * @private
  */
-export const participantListWrapper = mergeStyles({
-  width: '20rem',
-  // max width at 50% of view so the People Pane is not squeezing the Message Pane to almost nothing when on small screen or high zoom in
-  maxWidth: '50vw',
-  height: '100%'
-});
+export const participantListWrapper = (theme: Theme): string =>
+  mergeStyles({
+    width: '20rem',
+    // max width at 50% of view so the People Pane is not squeezing the Message Pane to almost nothing when on small screen or high zoom in
```

Comment:
This comment doesn't apply anymore since people pane is now overlayed on top of message thread.